cfme/tests/test_rest.py

```diff
@@ -371,31 +684,70 @@ def test_vm_add_lifecycle_event(request, rest_api, vm, from_detail, db):
     ))) == 1, "Could not find the lifecycle event in the database"
 
 
+@pytest.mark.parametrize(
+    "multiple", [False, True],
+    ids=["one_request", "multiple_requests"])
+def test_automation_requests(request, rest_api, automation_requests_data, multiple):
+    """Test adding the automation request
+     Prerequisities:
+        * An appliance with ``/api`` available.
+    Steps:
+        * POST /api/automation_request - (method ``create``) add request
+        * Retrieve list of entities using GET /api/automation_request and find just added request
+    Metadata:
+        test_flag: rest, requests
+    """
+
+    if "automation_requests" not in rest_api.collections:
+        pytest.skip("automation request collection is not implemented in this version")
+
+    if multiple:
+        requests = rest_api.collections.automation_requests.action.create(*automation_requests_data)
+    else:
+        requests = rest_api.collections.automation_requests.action.create(
+            automation_requests_data[0])
+
+    def _finished():
+        flag = False
+        for request in requests:
+            request.reload()
+            if request.status.lower() in {"Error"}:
+                pytest.fail("Error during automation request: `{}`".format(request.message))
+            flag = request.request_state.lower() in {"finished"}
```

Comment:
We have a single `flag` var to hold up to 5 request results.
Does this really work the way it should? What happens if the first 4 requests are not finished and the 5th is finished?

Id consider removing the `flag` completely and simply do:
```
def _finished():
    for request in requests:
        ...
        if request.request_state.lower() not in {'finished'}:
            return False
    return True
```